### DISCUSSION TITLE
SDG as Orientation

### DISCUSSION THREAD

**Discussion by joshcoon** [open-sdg/open-sdg#1736] [General]
Hi there, I am working in a project on migration and integration at the Statistical Office Germany. We would like to create an online-platform for our data. The SDG-platform is very intuitive and nicely composed. Is it possible to use some programming elements of SDG to create our platform? 
Best joshkoon

**Comment by christoph4711** [comment]
Have Look here https://christoph-lsn.github.io/MT_Site/

Von meinem iPhone gesendet

**Reply by joshcoon** [reply]
That´s great, thx!

**Reply by christoph-LSN** [reply]
@joshcoon don't hesitate to call me in my office tomorrow (Friday 10:00-18:00): (0511) 9898 2218
